docs/content/docs/user-docs/cross-account-resource-management.md

```diff
@@ -84,5 +84,4 @@ EOF
 ```
 
 ## Next Steps
-Checkout the [RBAC and IAM permissions overview](../authorization) to understand how ACK manages
-authorization
+Checkout the [RBAC and IAM permissions overview](../authorization) to understand how ACK manages authorization
```

Comment:
Just a dangling newline that I noticed. Can remove this change if anyone prefers typos fixes in separate PRs.